src/main/java/org/spongepowered/api/event/entity/ChangeEntityPotionEffectEvent.java

```diff
@@ -51,7 +51,22 @@
     /**
      * An event where the {@link PotionEffect} is being added.
      */
-    interface Gain extends ChangeEntityPotionEffectEvent { }
+    interface Gain extends ChangeEntityPotionEffectEvent {
+        
+        /**
+         * Gets the original potion effect involved in this event.
+         *
+         * @return The original potion effect involved in this event
+         */
+        PotionEffect getOriginalPotionEffect();
+        
+        /**
+         * Sets the potion effect to be used in this event.
```

Comment:
It doesn't matter; a custom value is a custom value, and it's only a single value.